File: 02 백준/08 정렬/Python/Prob2751.py

"""
title  : 2751. 수 정렬하기 2 (Silver 5)
time   : 1236ms       624ms
memory : 84788KB      137232KB
"""

import sys

def solution():
    input = sys.stdin.readline

    N = int(input())
    number = [int(input()) for _ in range(N)]

    for num in sorted(number):
        print(num)

solution()

File: 02 백준/08 정렬/Java/src/Prob2751.java

/**
* title  : 2751. 수 정렬하기 2 (Silver 5)
* time   : 1380ms
* memory : 108712KB
*/

import java.io.*;
import java.util.*;

public class Prob2751 {
    
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int N  = Integer.parseInt(br.readLine());

        int[] number = new int[N];
        for (int i = 0; i < N; i++) {
            number[i] = Integer.parseInt(br.readLine());
        }

        Arrays.sort(number);

        for (int i = 0; i < (N-1); i++) {
            sb.append(number[i]).append("\n");
        }
        
        sb.append(number[N-1]);

        System.out.println(sb);
    }
}
